browser.js

```diff
@@ -26,13 +32,30 @@ function randomBytes (size, cb) {
   }
 
   // XXX: phantomjs doesn't like a buffer being passed here
-  var bytes = Buffer.from(rawBytes.buffer)
+  return Buffer.from(rawBytes.buffer)
+}
+
+function randomBytes (size, cb) {
+  if (size > MAX_UINT32) {
+    throw new Error('Requested too many random bytes - the maximum allowed are ' + MAX_UINT32)
+  }
+
+  var bytes = Buffer.alloc(0)
```

Comment:
why are we making this empty buffer, can't we just make this undefined

Reply:
As you prefer.